experiments/krico/workloads/utils.go

```diff
@@ -0,0 +1,109 @@
+package workload
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/intelsdi-x/swan/pkg/conf"
+	"github.com/intelsdi-x/swan/pkg/executor"
+	"github.com/intelsdi-x/swan/pkg/experiment"
+	"github.com/libvirt/libvirt-go"
+	"github.com/pkg/errors"
+)
+
+const (
+	// TypeCaching is constant name of caching workload.
+	TypeCaching = "caching"
+)
+
+var (
+	snapStartCommand = "systemctl restart snap-telemetry"
+	aggressorAddress = conf.NewStringFlag("aggressor_address", "IP address of aggressor node.", "127.0.0.0")
```

Comment:
You do not launch any agressors.

Reply:
This flag must be renamed and moved to appropriate file: the one where it is used.

Reply:
But this flag would be used in many workloads that's why it is in utils

Reply:
In Swan "aggressor" is a synthetic workload that competes with high priority application for a resource. This is wrong place and wrong name, I believe.